src/main/java/Duke.java

```diff
@@ -1,10 +1,64 @@
+import duke.command.Command;
+import duke.exception.DukeException;
+import duke.parser.Parser;
+import duke.storage.Storage;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+import java.io.FileNotFoundException;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeFormatterBuilder;
+import java.util.*;
```

Comment:
The coding standard suggests to use single class imports instead of wildcard imports!